backend/promptfoo/agent_selection_config.yaml

```diff
@@ -253,3 +272,15 @@ tests:
     assert:
       - type: javascript
         value: JSON.parse(output).agent === "DatastoreAgent"
+
+
+  - description: "Test WHEN company file is uploaded and user has asked a question and web agent not available THEN FileAgent is selected"
```

Comment:
would the file agent be selected in this case if the web agent was still available? I'd imagine that File Agent would be preferred over the web agent